core/src/com/mygdx/minigolf/controller/EntityFactory.java

```diff
@@ -0,0 +1,48 @@
+package com.mygdx.minigolf.controller;
+
+import com.badlogic.ashley.core.Component;
+import com.badlogic.ashley.core.Engine;
+import com.badlogic.ashley.core.Entity;
+import com.mygdx.minigolf.model.EntityType;
+import com.mygdx.minigolf.model.components.Graphical;
+import com.mygdx.minigolf.model.components.Objective;
+import com.mygdx.minigolf.model.components.Physical;
+import com.mygdx.minigolf.model.components.Player;
+import com.mygdx.minigolf.model.components.PowerUpAffectable;
+import com.mygdx.minigolf.model.components.PowerUpGiver;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+
+public class EntityFactory {
+
+    private final Engine engine;
```

Comment:
Kjapt spm. Hva er planen for engine? Brukes ikke one sted for øyeblikket.

Reply:
nvm jeg er blind

Reply:
Entity factory bruker engine til Ashley ECS for å legge til entiteter til spillmotoren. Tenkte det var greit å legge dem til med en gang man lager dem, men kanskje vi skal gi det ansvaret til de som bruker EntityFactory?